app/src/main/java/com/github/palFinderTeam/palfinder/meetups/activities/MeetUpView.kt

```diff
@@ -122,6 +126,27 @@ class MeetUpView : AppCompatActivity() {
         }
     }
 
+    /**
+     * Clicking on QR Code icon will show QR code
+     */
+    fun showQR(view: View?) {
+        //Initiate the barcode encoder
+        val barcodeEncoder = BarcodeEncoder()
+        //Encode text in editText into QRCode image into the specified size using barcodeEncoder
+        //0 will be used for an user_id, 1 for a meetup id
```

Comment:
You should change this and use proper identification, or remove the 0 and 1, move the constant to the integers.xml and call them accordingly.